core/arch/arm/include/kernel/user_access_arch.h

```diff
@@ -18,4 +18,26 @@ static inline void enter_user_access(void) {}
 static inline void exit_user_access(void) {}
 #endif /* CFG_PAN */
 
+#ifdef ARM64
+#define get_user(_x, _p) ({						\
+	TEE_Result __res = TEE_SUCCESS;					\
+	__typeof(_x) __x = 0;						\
+	COMPILE_TIME_ASSERT(sizeof(_x) == sizeof(*_p));			\
+	__res = copy_from_user(&__x, (const void *)_p, sizeof(__x));	\
+	(_x) = (__x);							\
+	__res;								\
+})
+
+#define put_user(_x, _p) ({						\
+	TEE_Result __res = TEE_SUCCESS;					\
+	__typeof(*(_p)) __x = (_x);					\
+	COMPILE_TIME_ASSERT(sizeof(_x) == sizeof(*_p));			\
```

Comment:
s/*_p/*(_p)/
use `static_assert()` instead of `COMPILE_TIME_ASSERT()`.
add an empty line after local variable definition.

By the way, @jforissier, @jenswi-linaro, would you be ok we create a P-R to remove `COMPILE_TIME_ASSERT()`?

Reply:
switched to `static_assert()`